horace_core/sqw/@sqw/slim.m

```diff
@@ -13,6 +13,12 @@
 % -------
 %   wout    Output sqw object or array of szqw objects
 
-wout = mask_pixels_random_fraction(win, 1/reduce);
+if reduce<1
+    error('HORACE:sqw:invalid_argument', ...
+        'reduce factor has to be bigger then 1. It is %s', ...
```

Comment:
Message does not match condition. 
```suggestion
        'reduce factor has to be bigger than 0. It is %s', ...
```
or 
```
if reduce <= 1
```

Reply:
no, 1. 1 -- all remains more then 1 -- appropriate fraction removed.

Reply:
In this case `1.0` would be accepted which presumably is not what you want?

Reply:
1 will leave all pixels which makes no sence but may be useful for some algorithms